datalad/interface/tests/test_run_procedure.py

```diff
@@ -339,3 +340,18 @@ def test_quoting(path):
             "datalad run-procedure just2args \"with ' sing\" 'with \" doub'")
         with assert_raises(CommandError):
             runner.run("datalad run-procedure just2args 'still-one arg'")
+
+@skip_if_on_windows
+@with_tempfile
+def test_text2git_empty(path):
+    """
+    Tests that empty files are not annexed in a ds configured with text2git.
+    """
+    ds = Dataset(path).create(force=True)
+    ds.run_procedure('cfg_text2git')
+    ok_clean_git(ds.path)
+    # create an empty file, no extension
+    open(op.join(path, 'emptyfile'), 'a').close()
```

Comment:
Hmm, the 'a' rather than 'w' made me pause.  We're in a new empty directory though, so either way is fine.

Just a note: `@with_tree` is nice for pre-populating the test directory with files and contents.  In this case, I don't using it really makes the test any more readable (and would make the `ok_clean_git` call more complicated).